contracts/wallet.sol

```diff
@@ -460,15 +460,15 @@ contract LoadLimit is ControllableOwnable, SelfCallableOwnable {
     /// @dev Sets a daily card load limit.
     /// @param _amount is the card load amount in current stablecoin base units.
     function setLoadLimit(uint _amount) external onlyOwnerOrSelf {
-        require(_MINIMUM_LOAD_LIMIT <= _amount && _amount <= _maximumLoadLimit, "out of range load amount");
+        require(_minimumLoadLimit <= _amount && _amount <= _maximumLoadLimit, "out of range load amount");
```

Comment:
`_minimumLoadLimit` is guaranteed to be zero now so we are checking if an unsigned integer is <= to zero which is always true.  So we can take out the `_minimumLoadLimit <= _amount &&` part.